docling_mcp/servers/mcp_server.py

```diff
@@ -17,7 +17,8 @@ class ToolGroups(str, enum.Enum):
     CONVERSION = "conversion"
     GENERATION = "generation"
     MANIPULATION = "manipulation"
-    LLAMA_INDEX_RAG = "llama-index-rag"
+    LLAMA_INDEX_RAG = "li-rag"
+    LLAMA_STACK_RAG = "lls-rag"
```

Comment:
sorry in advance for my ocd: why does one have 2 ll's and the other only 1?

Reply:
Well, it is how these libraries are usually shortened.
We could go with `lli`, or keep the full string.

In general, if we think of other frameworks, e.g. langchain, there won't be an homogeneous shortening.